### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by FedeOt**
-Adds redux to checking accounts visualization.
-Adds testing files to check if redux is working fine.

### CODE REVIEW

File: src/redux/accountSlice.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    },
+    reducers:{
+        getAccounts(state,action){
+            state.accounts = action.payload; 
+        }
+    }
+}); 
+
+export const {getAccounts} = accountSlice.actions;
+export default accountSlice.reducer;
```
Comment (ghost) [inline src/redux/accountSlice.js]: I'd rename this file to `accountReducer.js` as _slice_ - even though it comes from the Redux Toolkit package - isn't too intuitive at first.

File: src/tests/Redux.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      store.dispatch(getAccounts(testingPayload));
+      const state = store.getState().accounts;
+      expect(state.accounts).toEqual(testingPayload);   
+      
+    });
+    
+
+
+  });
```
Comment (ghost) [inline src/tests/Redux.test.js:42]: Could you format this file? :)

**Review by ghost — CHANGES_REQUESTED** [review]
I really liked how Redux was implemented in your project! It looks clean and it's amazing how the Redux Toolkit reduces configuration lines, the need of installing extra dependencies, etc.

Excellent job! :)

File: src/redux/accountSlice.js (hunk 1 above)
Comment (FedeOt) [reply src/redux/accountSlice.js]: You are right, I'll rename it

File: src/tests/Redux.test.js (hunk 2 above)
Comment (FedeOt) [reply src/tests/Redux.test.js:42]: Sure!

**Review by ghost — APPROVED** [review]

**Pull request merged**